apps/condo/domains/notification/constants/constants.js

```diff
@@ -52,6 +53,7 @@ const MESSAGE_TYPES = [
     TICKET_COMMENT_ADDED_TYPE,
     BILLING_RECEIPT_AVAILABLE_MANUAL_TYPE,
     BILLING_RECEIPT_AVAILABLE_NO_ACCOUNT_MANUAL_TYPE,
+    RESIDENT_ADD_ACCOUNT_MANUAL_TYPE,
```

Comment:
PLEASE STOP ADDING MANUAL SUFFIXES! You should use the same messages for non-manual messages. Don't multiply messages.

Reply:
fixed